git/test/lib/helper.py

```diff
@@ -139,7 +139,7 @@ def repo_creator(self):
             try:
                 try:
                     return func(self, rw_repo)
-                except:
+                except:  # noqa E722
```

Comment:
How about `except Exception:` like elsewhere?

Reply:
We could. But at this out-most level of handling execution, I thought to remain as "greedy" as possible.  May be someone (among used dependencies) still manages to raise non-Exception "exceptions" up the chain from somewhere/somehow -- so I thought to remain catching them as well.  In the other places it was more "specific" to not expect some complete weird stuff being thrown up.  But if you insist, we could have it Exception here as well